src/services/generate-cv-service.js

```diff
@@ -0,0 +1,11 @@
+import { generatepdfEndpoint } from './config';
+
+export const fetchCV = () =>
+  fetch(generatepdfEndpoint, {
```

Comment:
I'd change the filename to be generateCVService.js and if firing multiple requests off maybe set up the headers in a utility sendAPIRequest func or something along those lines so you don't set the headers for each service func.

Reply:
@girwin90 is camel case the preferred convention for filenames here? I have used kebab  case throughout the project so if so I will make the change everywhere.

Reply:
I usually stick to camelCase myself, most projects I work on do also.